pages/UI.php

```diff
@@ -1852,8 +1852,9 @@ function DisplayNavigatorGroupTab($oP)
 		break;
 
 		///////////////////////////////////////////////////////////////////////////////////////////
-		
-		case 'swf_navigator': // Graphical display of the relations "impact" / "depends on"
+
+		case 'swf_navigate': /** @deprecated SWF was removed in iTop */
```

Comment:
Did you mean "swf_navigat**or**"?

```suggestion
		case 'swf_navigator': /** @deprecated SWF was removed in iTop */
```

Reply:
Ouch, nice catch Guillaume !

Reply:
Whoops indeed. Came from e0079a41e5df4bd106441ca518a107bdd31330da